spec/validator_spec.js

```diff
@@ -4,8 +4,8 @@ const fs = require("fs");
 const path = require("path");
 const validator = require("../src/validator");
 
-function validate(xmlData, error, line = 1) {
-    const result = validator.validate(xmlData);
+function validate(xmlData, ignoreNameSpace, error, line = 1) {
```

Comment:
instead of `ignoreNameSpace` flag we can pass options to reuse the function when any other option is changed.

Reply:
Done!